### PR TITLE
fix(segmentedcontrol) color and selectors

### PR DESCRIPTION
**Pull request by jackkav**
### Summary
Before:
image

After:
image

#### Changes made:
* mute colors to suit konnect design
* fix hover active and focus borders and box shadows

NOTE: in previous version hover focus and active states were all handled gracefully by kbutton, in this design they have to be overriden
image

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: packages/KSegmentedControl/KSegmentedControl.vue [hunk 1]
```diff
@@ -68,24 +68,41 @@ export default {
...
-  margin-left: 0;
-}
-.segmented-control .k-button:last-child {
-  border-radius: 0 3px 3px 0;
-}
-.segmented-control .k-button:only-child {
-  border-radius: 3px;
-  margin-left: 0;
+  color: var(--blue-600);
+  --KButtonPrimaryBase: var(--blue-100);
```
Comment (travispamaral) [inline packages/KSegmentedControl/KSegmentedControl.vue:77]: Let's move these to the top of block. And then lets use a new var with a fallback of blue-600 and import the vars so like in other kongponents.

```scss
@import '~@kongponents/styles/_variables.scss';

.segmented-control .k-button {
--KButtonPrimaryBase: var(--blue-100);
--KButtonPrimaryHover: var(--blue-100);

color: var(--KSegementedControlPrimary, var(--blue-600, color(blue-600)));
```

File: packages/KSegmentedControl/KSegmentedControl.vue [hunk 2]
```diff
@@ -68,24 +68,41 @@ export default {
...
+    z-index: 1;
+    border-color: var(--blue-600);
+  }
+  &:hover {
+    z-index: 2;
+    border-color: var(--blue-600);
+  }
+  &:active {
+    z-index: 2;
+    border-color: var(--blue-600);
```
[the remark this answers is not in the document]
Comment (travispamaral) [reply packages/KSegmentedControl/KSegmentedControl.vue:89]: these can all be the same zindex too I think

File: packages/KSegmentedControl/KSegmentedControl.vue [hunk 3]
```diff
@@ -68,24 +68,41 @@ export default {
...
+  }
+  &:focus {
+    z-index: 3;
+    border-color: var(--blue-600);
```
Comment (travispamaral) [inline packages/KSegmentedControl/KSegmentedControl.vue:93]:
```suggestion
```

**Review by travispamaral — CHANGES_REQUESTED** [review]

**Comment by jackkav** [comment]
focus is z-index 3 to avoid this case 
image

**Pull request merged**